.github/workflows/integration-tests.yml

```diff
@@ -65,7 +65,7 @@ jobs:
                 image: ghcr.io/balhar-jakub/gateway-service:${{ github.run_id }}-${{ github.run_number }}
                 env:
                     APIML_SECURITY_AUTH_JWT_CUSTOMAUTHHEADER: customJwtHeader
-                    APIML_GATEWAY_HEALTH_PROTECTED: true
+                    APIML_HEALTH_PROTECTED: true
```

Comment:
Default is false, this could be one of the examples.

https://github.com/zowe/api-layer/blob/3346a98e5f2bd56b4d00947af0da607c1805b56e/integration-tests/src/test/java/org/zowe/apiml/functional/discovery/DiscoveryServiceAuthenticationTest.java#L85